Source/Core/Core/PowerPC/CachedInterpreter/CachedInterpreter.cpp

```diff
@@ -210,6 +211,15 @@ void CachedInterpreter::WriteBrokenBlockNPC(CachedInterpreter& cached_interprete
   cached_interpreter.m_ppc_state.npc = data.hex;
 }
 
+void CachedInterpreter::DoBranchWatchHit(CachedInterpreter& cached_interpreter, UGeckoInstruction n)
+{
+  if (auto& branch_watch = cached_interpreter.m_branch_watch; branch_watch.GetRecordingActive())
+  {
+    const PPCAnalyst::CodeOp& op = cached_interpreter.m_code_buffer[n.hex];
```

Comment:
I have now realized that the Cached Interpreter JIT doesn't have OPTION_BRANCH_FOLLOW enabled, and since that is the only reason for this function to exist, it can be removed.

Reply:
That's true! It should be fine to change the `op.skip` case to just an assert that triggers on any instruction.